omrmakefiles/rules.zos.mk

```diff
@@ -36,7 +36,6 @@ endif
 
 # Enable Debugging Symbols
 ifeq ($(OMR_DEBUG),1)
-  GLOBAL_FLAGS+=-Wc,g9
 endif
```

Comment:
This `ifeq` is now empty. Can it be folded away?

Reply:
I was leaving it the same as the 2 above so if we figure out the right options we can add them in.  I can remove it / them if you prefer

Reply:
Oh I see. I didn't notice the ones above. I was just curious. I vote to stay consistent then and keep the change as is since `OMR_WARNINGS_AS_ERRORS` and  `OMR_ENHANCED_WARNINGS` `ifeq` checks are empty as well. Thanks for the clarification!